charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/kube-apiserver.rules.test.yaml

```diff
@@ -82,4 +87,16 @@ tests:
       exp_annotations:
         description: Kube API server latency for verb POST is high. This could be because the shoot workers and the control plane are in different regions. 99th percentile of request latency is greater than 3 second.
         summary: Kubernetes API server latency is high
-
+  - eval_time: 16m
+    alertname: KubeApiServerTooManyAuditlogFailures
```

Comment:
Thanks for writing the test! Unfortunately, this test does not pass. Run `make verify` to see if the test passes or reference the [monitoring docs](https://github.com/gardener/gardener/blob/master/docs/development/monitoring-stack.md#alert-tests) on using `promtool`.